client/my-sites/site-settings/form-general.jsx

```diff
@@ -133,11 +135,23 @@ class SiteSettingsFormGeneral extends Component {
 
 		if (
 			this.props.isSavingSettings &&
-			! nextProps.isSavingSettings &&
-			nextProps.isSaveRequestSuccessful
+			! nextProps.isSavingSettings
 		) {
-			nextProps.clearDirtyFields();
-			nextProps.markSaved();
+			if ( nextProps.isSaveRequestSuccessful ) {
+				nextProps.successNotice( nextProps.translate( 'Settings saved!' ), { id: 'site-settings-save' } );
+				nextProps.clearDirtyFields();
+				nextProps.markSaved();
+			} else {
+				let text;
+				switch ( nextProps.saveRequestError.error ) {
+					case 'invalid_ip':
+						text = nextProps.translate( 'One of your IP Addresses was invalid. Please try again.' );
```

Comment:
I don't really know, I searched and found the error key on the API side but I'm not sure how to reproduce. 

This is error was here in the first place, that's why I did not touch it, but maybe worth removing since I don't think the user wants to know about this. We could even drop the "error" and the corresponding selector from the redux state. What do you think?